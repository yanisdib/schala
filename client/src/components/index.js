// COMPONENTS EXPORTS

export { default as Header } from './Header/Header';

export { default as Button } from './Button/Button';
export { default as MenuButton } from './MenuButton/MenuButton';
export { default as DisplayIdentifiedButton } from './DisplayIdentifiedButton/DisplayIdentifiedButton';

export { default as Tag } from './Tag/Tag';
export { default as Pointer } from './Pointer/Pointer';
export { default as Bubble } from './Bubble/Bubble';

export { default as PictureIdentifier } from './PictureIdentifier/PictureIdentifier';