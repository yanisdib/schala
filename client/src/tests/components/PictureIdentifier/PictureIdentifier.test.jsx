import { shallow } from 'enzyme';

import { PictureIdentifier } from '../../../components';


describe('PictureIdentifier component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<PictureIdentifier />))

    it('renders correctly', () => expect(wrapper).toMatchSnapshot())
})