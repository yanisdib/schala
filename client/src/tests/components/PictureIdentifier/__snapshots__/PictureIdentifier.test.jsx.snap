// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PictureIdentifier component renders correctly 1`] = `
<Fragment>
              
  <img
    alt=""
    onLoad={[Function]}
    src="https://pbs.twimg.com/media/EpjlsCFW8AMJtO1.jpg"
    title=""
  />
  <DisplayIdentifiedPeopleButton
    onClick={[Function]}
  />
</Fragment>
`;
